mrp_bom_version/models/res_config.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
+# (c) 2015 Oihane Crucelaegui - AvanzOSC
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+from openerp import api, fields, models
+
+
+class MrpConfigSettings(models.TransientModel):
+    _inherit = 'mrp.config.settings'
+
+    group_mrp_bom_version = fields.Boolean(
+        string='Allow to re-edit BoMs',
+        implied_group='mrp_bom_version.group_mrp_bom_version',
+        help='The active state may be passed back to state draft')
+    active_draft = fields.Boolean(
+        string='Keep re-editing BoM active',
+        help='This will allow you to define if those BoM passed back to draft'
```

Comment:
This will allow you to define whether those BoM passed back to draft
can still be re-activated or not.